### PR TITLE
Conner/daily fetch

### PR DESCRIPTION
**Pull request by ilikesymmetry**
Added a feature that allows the server to get and parse Cornell Dining's JSON and update our graphql db every set time period.  The current set time period is 24 hours.

### CODE REVIEW

File: constants.py [hunk 1]
```diff
@@ -0,0 +1 @@
+UPDATE_DELAY = 86400 #24hrs
```
Comment (young-k) [inline constants.py]: 24 hours in milliseconds (make the comment more clear), also there should be a space after the `#`, so `# 24`

**Review by young-k — APPROVED** [review]
lgtm, make changes before merging

File: data.py [hunk 2]
```diff
@@ -1,8 +1,10 @@
 # Will update the graphQL db by calling scraper.py every defined time interval
 from datetime import datetime
-import requests
+from requests import get
 import schema
 from bs4 import BeautifulSoup
+from threading import Timer
```
Comment (young-k) [inline data.py:6]: alphabetize these imports, so `bs4` at the top, ... etc

**Review by young-k — CHANGES_REQUESTED** [review]
jk, one more round

**Comment by ilikesymmetry** [comment]
@young-k

**Review by young-k — APPROVED** [review]
In the future, make sure to change the pull request title (should be something like `Added daily fetch...`).

Also rebase away the last two commits before merging!

**Pull request merged**